pkg/parquetquerygen/predicates.go

```diff
@@ -90,6 +90,110 @@ func (p {{ $structName }}) KeepValue(v pq.Value) bool {
 }
 {{- end }}
 {{- end }}
+
+{{- /* Generate In/NotIn for Int, Float, Bool */}}
```

Comment:
predicates gen is a nice perf boost over the generic predicate we had before, but i still hate it. do you think we can do this with go generics?

i don't expect that change in this PR. just wondering.